pasqal_cloud/_version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.1.14"
+__version__ = "0.1.17"
```

Comment:
As required by @HGSilveri in #73 we should try to stick to semantic versioning. This would effectively act as a breaking change and hence cannot be bumped a minor version. This should be at least 0.2.0, maybe it's even time for 1.0.0